marconi-chain-index/test-lib/Gen/Marconi/ChainIndex/Types.hs

```diff
@@ -185,7 +185,20 @@ genTxBodyContentWithTxInsCollateral
   -> Gen (C.TxBodyContent C.BuildTx era)
 genTxBodyContentWithTxInsCollateral era txIns txInsCollateral = do
   txbody <- CGen.genTxBodyContent era
-  txProtocolParams <- C.BuildTxWith . Just <$> CGen.genProtocolParameters
+  initialPP <- CGen.genProtocolParameters C.BabbageEra
+  let txProtocolParams =
+        C.BuildTxWith $
+          Just $
+            initialPP
+              { C.protocolParamUTxOCostPerWord = Just 1
+              , C.protocolParamUTxOCostPerByte = Just 1
+              , C.protocolParamPrices = Just $ C.ExecutionUnitPrices 1 1
+              , C.protocolParamMaxTxExUnits = Just $ C.ExecutionUnits 1 1
+              , C.protocolParamMaxBlockExUnits = Just $ C.ExecutionUnits 1 1
+              , C.protocolParamMaxValueSize = Just 1
+              , C.protocolParamCollateralPercent = Just 1
+              , C.protocolParamMaxCollateralInputs = Just 1
+              }
```

Comment:
It looks like these parameters are set in `cardano-api-gen:8.1.1.0` did you try to update it? https://github.com/input-output-hk/cardano-api/blob/cardano-api-gen-8.1.1.0/cardano-api/gen/Test/Gen/Cardano/Api/Typed.hs

Reply:
Actually, `cardano-api-gen-8.1.1.0` is empty. The link you provided is from cardano-api. So yeah, I'm using cardano-api-8.8 and these fields are still missing. Removing them makes the tests fail.

Reply:
Oh, so we need to amend this once `cardano-api-gen` is updated then. :(